saos-common/src/main/java/pl/edu/icm/saos/common/CommonConfiguration.java

```diff
@@ -41,8 +41,8 @@ public MappingJsonFactory jsonFactory() {
     }
     
     @Bean
```

Comment:
Well, I thought about possible future movement of this or other common classes from saos-common to icm-common and didn't want to have the class bound to spring. But... I've changed it now cause I think we shouldn't think to much about the possible future usage and cause we use spring in other projects as well.